lib/gollum/app.rb

```diff
@@ -22,6 +22,13 @@
 # Fix to_url
 class String
   alias :upstream_to_url :to_url
+
+  unless defined?(Grit)
```

Comment:
That's not possible at the moment: although we can check whether `GIT_ADAPTER != 'grit'`, that doesn't ensure that gollum-lib will not fall back on grit anyway (when `GIT_ADAPTER` cannot be required). See [here](https://github.com/gollum/gollum-lib/blob/master/lib/gollum-lib.rb#L9-L13).